server/test/handlers/getWorkersForSchool.test.js

```diff
@@ -0,0 +1,121 @@
+const schoolHandler = require('../../models/handlers/school');
+
+describe('testing invalid schoolId parameters', () => {
+    beforeEach(() => {
+        jest.resetModules(); // Clears any cache between tests.
+    });
+
+    test('rejection of no parameters', async () => {
+        // Arrange
+        // No test variables need to be initialized.
+
+        // Act
+        const isSuccessfullyInserted = await schoolHandler.getWorkerIdsForSchool();
```

Comment:
Hi Simran,
Please take another look at some of the stub PRs (e.g. #21) as the initial test should ensure that the workerIds returned form the schoolHandler method is the expected response. Thus, I would comment out all the code and leave a comment that it will be useful when testing the logic of the endpoint. Then, I would create an initial test that just tests to make sure that the values returned from the `getWorkerIdsForSchool` match `[8000000, 8000001, 8000002, 8000003]` (which can be stored in a variable).

Reply:
Hi Amy! I've attempted to do something. Please let me know what you think! 

Thanks!